content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md

```diff
@@ -75,6 +75,11 @@ only the termination message:
 
         kubectl get pod termination-demo -o go-template="{{range .status.containerStatuses}}{{.lastState.terminated.message}}{{end}}"
 
+ If you are running a multi-container pod, you can use a Go template to include the container's name. By doing so, you can discover which of the containers is failing:
+
+        kubectl get pod multi-container-pod -o go-template='{{range .status.containerStatuses}}{{printf "%s:\n%s\n\n" .name .lastState.terminated.message}}{{end}}'
```

Comment:
Please consider using explicit code snippet markup, i.e. the backtiqs rather than indentation.

Reply:
I am just following the rest of the document's convention. I think that changing this must be another's PR scope, to change them _all_? Changing only one seems weird.

Reply:
IMO not a blocking change but +1 to @tengqm - use a code fence for your change. Changing the existing content can be a different PR, but please follow our [contributor style guidelines](https://kubernetes.io/docs/contribute/style/style-guide/#code-style-inline-code) for new changes.

Reply:
Perfect